src/checks/checks-common.ts

```diff
@@ -84,18 +108,20 @@ export function printSummary(checkResult: CheckRunResult, ci?: boolean): void {
     return
   }
 
-  console.log(output.summary)
-  const annotations = output.annotations || []
-  for (const annotation of annotations) {
-    const { annotation_level, message, start_line, end_line, path } = annotation
-    let location = ''
+  if (output) {
+    console.log(output.summary)
+    const annotations = output.annotations || []
+    for (const annotation of annotations) {
+      const { annotation_level, message, start_line, end_line, path } = annotation
+      let location = ''
 
-    if (path) {
-      const lines = start_line && end_line ? ` line ${start_line}:${end_line}` : ''
-      location = `(${path}${lines})`
-    }
+      if (path) {
+        const lines = start_line && end_line ? ` line ${start_line}:${end_line}` : ''
```

Comment:
since `start_line` and `end_line` are not optional anymore, this can be simplified.